oai_agents/agents/rl.py

```diff
@@ -338,9 +353,9 @@ def train_agents(self, total_train_timesteps, tag, exp_name=None):
                     best_path, best_tag = self.save_agents(tag=KeyCheckpoints.BEST_EVAL_REWARD)
                     print(f'New best evaluation score of {mean_reward} reached, model saved to {best_path}/{best_tag}')
                     self.best_score = mean_reward
-            steps += 1
+            self.steps += 1
         self.save_agents()
```

Comment:
We need tag=KeyCheckpoints.MOST_RECENT_TRAINED_MODEL to make sure we save it using this tag since we are using it in all adversary-play functions.

Reply:
suggested change:
self.save_agents(tag=KeyCheckpoints.MOST_RECENT_TRAINED_MODEL)

Reply:
I thought it was saving under the checkpoint name and not last, last is supposed to be the final ckpt? @avv-va maybe you can clarify

Reply:
We save agents using three ways: 
1. checkpoints throughout the training which are saved under ck_rew_...dir
2. best agent that is being updated throughout the training and is saved under best dir
3. the last trained checkpoint which is only saved if the agent finishes training. it is saved under last dir (previously aamas25)

To fix this, tag should be passed to self.save_agents, I thought I commited this to your remove_aamas PR but apparently I forgot it. (It doesn't mean that our experiments were wrong so far because the default in save_agents is last). Can you change this to as chihui mentioned: 
`self.save_agents(tag=KeyCheckpoints.MOST_RECENT_TRAINED_MODEL)`